### PR TITLE
feat(appstream): Service and Checks

### PR DESCRIPTION
**Pull request by jfagoagas**
### Context 

We have to migrate the Prowler v2 checks to the v3.

### Description

- AppStream 2.0 Service
- appstream_fleet_default_internet_access_disabled | extra7193
- appstream_fleet_maximum_session_duration | extra7190
- appstream_fleet_session_disconnect_timeout | extra7191
- appstream_fleet_session_idle_disconnect_timeout | extra7192

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: providers/aws/services/accessanalyzer/accessanalyzer_service_test.py [hunk 1]
```diff
@@ -65,6 +65,11 @@ def test__get_session__(self):
         access_analyzer = AccessAnalyzer(current_audit_info)
         assert access_analyzer.session.__class__.__name__ == "Session"
 
+    # Test AccessAnalyzer Session
```
Comment (MrCloudSec) [inline providers/aws/services/accessanalyzer/accessanalyzer_service_test.py]:
```suggestion
    # Test AccessAnalyzer Service
```

File: providers/aws/services/appstream/appstream_fleet_session_idle_disconnect_timeout/appstream_fleet_session_idle_disconnect_timeout.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.appstream.appstream_client import appstream_client
+
+max_idle_disconnect_timeout_in_seconds = 10 * 60
+"""max_disconnect_timeout_in_seconds 600"""
```
Comment (MrCloudSec) [inline providers/aws/services/appstream/appstream_fleet_session_idle_disconnect_timeout/appstream_fleet_session_idle_disconnect_timeout.py]: How about setting this variable in config.yaml and get it with get_config_var function?

File: providers/aws/services/appstream/appstream_fleet_session_disconnect_timeout/appstream_fleet_session_disconnect_timeout.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.appstream.appstream_client import appstream_client
+
+max_disconnect_timeout_in_seconds = 5 * 60
+"""max_disconnect_timeout_in_seconds 300"""
```
Comment (MrCloudSec) [inline providers/aws/services/appstream/appstream_fleet_session_disconnect_timeout/appstream_fleet_session_disconnect_timeout.py]: How about setting this variable in config.yaml and get it with get_config_var function?

File: providers/aws/services/appstream/appstream_fleet_maximum_session_duration/appstream_fleet_maximum_session_duration.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.appstream.appstream_client import appstream_client
+
+max_session_duration_seconds = 10 * 60 * 60
+"""max_session_duration_seconds 36000"""
```
Comment (MrCloudSec) [inline providers/aws/services/appstream/appstream_fleet_maximum_session_duration/appstream_fleet_maximum_session_duration.py]: How about setting this variable in config.yaml and get it with get_config_var function?

**Review by MrCloudSec — CHANGES_REQUESTED** [review]
Way to go 🔝 pls, check my comments

File: providers/aws/services/appstream/appstream_fleet_session_disconnect_timeout/appstream_fleet_session_disconnect_timeout.py (hunk 3 above)
Comment (jfagoagas) [reply providers/aws/services/appstream/appstream_fleet_session_disconnect_timeout/appstream_fleet_session_disconnect_timeout.py]: Done!

File: providers/aws/services/appstream/appstream_fleet_session_idle_disconnect_timeout/appstream_fleet_session_idle_disconnect_timeout.py (hunk 2 above)
Comment (jfagoagas) [reply providers/aws/services/appstream/appstream_fleet_session_idle_disconnect_timeout/appstream_fleet_session_idle_disconnect_timeout.py]: Done!

File: providers/aws/services/appstream/appstream_fleet_maximum_session_duration/appstream_fleet_maximum_session_duration.py (hunk 4 above)
Comment (jfagoagas) [reply providers/aws/services/appstream/appstream_fleet_maximum_session_duration/appstream_fleet_maximum_session_duration.py]: Done!

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**